R/qenv-get_env.R

```diff
@@ -0,0 +1,32 @@
+#' Extract Environment Included in `qenv`
+#'
+#' The extraction of environment included in `qenv@env` allows to list object names included in `qenv@env` slot.
+#'
+#' @param object (`qenv`)
+#'
+#' @return A `character` vector with names of objects.
```

Comment:
nope

Reply:
I wrote a documentation for ls() and then decided to leave it as get_env : D 4e8aa987d279ce694b657eb5ee212191f7f440fb thanks - fixed